### DISCUSSION TITLE
modern.runtime.ts中的defineRuntimeConfig的作用是什么？是否可以补充一下文档呢

### DISCUSSION THREAD

**Discussion by yunloon** [web-infra-dev/modern.js#6213] [General]
modern.runtime.ts中的defineRuntimeConfig的作用是什么？字面理解是配置运行时的相关设置，但是在官网的文档中搜索不到相关的内容以及具体的用法，可否补充一下文档呢

**Comment by lanmingle** [comment]
+1

**Comment by zllkjc** [comment]
这个还是个内测功能。应该会在 10 月会去开放，然后补充上去。目前暂时先把模板透出来了，可以尝试去添加运行时插件

**Comment by zllkjc** [comment]
配置文件：https://modernjs.dev/zh/configure/app/runtime/0-intro.html
运行时插件：https://modernjs.dev/zh/plugin/introduction.html#runtime-%E6%8F%92%E4%BB%B6

已经补充文档。
